### DISCUSSION TITLE
Why does Quickstrom fail with a page reload?

### DISCUSSION THREAD

**Discussion by mdeane-r7** [quickstrom/quickstrom#108] [Q&A] [answered]
Hi, as the above states I'm looking to find out more information on why quickstrom tests fail when a page reloads? As I've noticed recently in my application that the method `location.reload();` causes failures.

Thanks

**Comment by owickstrom** [comment, marked as answer]
I haven't tested any page that does this, but I do have some ideas about what could go wrong.

Would you be able to provide a minimal failing example? Something like the integration tests here: https://github.com/quickstrom/quickstrom/tree/main/integration-tests/failing

Thanks!

**Reply by mdeane-r7** [reply]
Yes I should be able to, I'll have to come up with something first but I'll reply to this when I have

**Reply by mdeane-r7** [reply]
I was unable to make a reproducible example, so I looked through my purs file again and found the issue stemmed from that so there seems to be no issue with page reload.
Thank you for your response :) !
